docker/scripts/p4runtime_lib/simple_controller.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python2
```

Comment:
I'm actually starting to doubt that this file is used anywhere?

Reply:
No. It was copied in from p4lang/tutorials. I removed the file.